consensus/ising/voting/sigchainvotingpool.go

```diff
@@ -80,34 +71,65 @@ func (scvp *SigChainVotingPool) AddToSendPool(height uint32, nid uint64, hash Ui
 	}
 }
 
-func (scvp *SigChainVotingPool) VoteCounting(height uint32) (*Uint256, error) {
-	scvp.RLock()
-	defer scvp.RUnlock()
+func (scvp *SigChainVotingPool) AddToReceivePool(height uint32, nodeID uint64, weight int, hash Uint256) {
```

Comment:
Why we need two methods addToReceivePool and AddToReceivePool instead of just AddToReceivePool?

Reply:
Both AddToReceivePool and AddVoteThenCounting need to add vote to pool. Function addToReceivePool is common function for them.

Reply:
so could we rename them to avoid confuse?

Reply:
Maybe we could make this to be common coding style.
For example, 
function InitTest() can call initTest()，
InitTest() is for public use and initTest() for internal use only.

Reply:
I think we can do it